### PR TITLE
Stop putting year in the copyright notice

### PR DESCRIPTION
**Pull request by howardjohn**
This was already done on a few files by Martin, so I assume he got legal
approval here.

The motivation with this is to end bikeshedding on PRs about the
copyright year.

### CODE REVIEW

**Review by nmittler — APPROVED** [review]

**Review by ramaraochavali — APPROVED** [review]

**Comment by rcaballeromx** [comment]
@howardjohn I would verify that approval. AFAIK copyright notices require the year save very few exceptions, see https://www.copyrightlaws.com/copyright-symbol-notice-year/

**Review by myidpt — APPROVED** [review]

**Review by bianpengyuan — APPROVED** [review]

**Review by elfinhe — APPROVED** [review]

**Comment by ericvn** [comment]
I did see this after a quick search: https://www.linuxfoundation.org/blog/2020/01/copyright-notices-in-open-source-software-projects/

That said. I'm not sure if they, the years, should continue to be updated or not.

**Review by costinm — APPROVED** [review]
I don't know - if reviewers don't have to complain about the year - they may start looking at code... 
It can be dangerous.

**Review by sdake — APPROVED** [review]
@howardjohn Standard disclaimer applies, I am not legal counsel, every situation is unique, and as a result, it is best to consult an attorney.

Here is the [header guidance](https://wiki.openstack.org/wiki/LegalIssuesFAQ#Copyright_Headers) I'd run with which involved hundreds of attorneys from `$$BIGCO$$s`.

if you want more information on how OpenStack came to this conclusion, OpenStack has a dynamic legal mailing list filled with armchair attorneys and, also, people that understand why the above guidance was created.

TLDR copyright notices are **NOT** required at all, however, if you wish to include them, adding multiple companies or years is highly confusing (as evidenced by the interest in seeing this merge). Guidance for web content differs from source code headers.

I know we had a script that automated the copyright generation. Was that modified here? I didn't actually look through all 1880 files :)
